code/iaas/logic/src/main/java/io/cattle/platform/process/instance/InstancePreCreate.java

```diff
@@ -95,7 +95,7 @@ protected void setName(Instance instance, Map<Object, Object> data) {
         if (instance.getName() != null) {
             return;
         }
-        String name = "r-" + instance.getUuid().substring(0, 4);
+        String name = "r-" + instance.getUuid().substring(0, 10);
```

Comment:
is this change unrelated to the serviceIndex issue? just want to make sure i understand